src/pages/Information/Information.jsx

```diff
@@ -1,9 +1,15 @@
 import React from "react";
+import { FormattedMessage } from "react-intl";
 
 const Information = () => (
   <>
     <div data-test="information-display">
-      <h2 data-test="heading-display">What is HacktoberFest?</h2>
+      <h2 data-test="heading-display">
+        <FormattedMessage
+          id="Info.first-header"
+          defaultMessage="What is HacktoberFest?"
+        />
+      </h2>
       <p data-test="para-display">
         Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```

Comment:
♻️ Let's add a translation for this, and any other hard coded text.

Reply:
Addressed:  6b7a9fae42ca32647bed94da74d304b11d5638f9